input/pagecontent/group-disease.md

```diff
@@ -37,15 +39,15 @@ Body locations in FHIR are typically represented using a single code. However, a
 
 mCODE has adopted an approach that allows the user to add additional code or codes to further define the body site, without the need to create an independent resource. This takes the form of the [LateralityQualifier] and [BodyLocationQualifier] extensions. These extensions can be used to specify laterality, directionality, and plane.
 
-### Staging
+### Staging and Risk Assessments
```

Comment:
If the modeling of Staging and Risk Assessments are supposed to be analogous/parallel, I'm not convinced that the FHIR profiles are analogous/parallel.